contracts/ColonyTask.sol

```diff
@@ -310,15 +310,16 @@ contract ColonyTask is ColonyStorage, DSMath {
     Task storage task = tasks[_id];
     IColonyNetwork colonyNetworkContract = IColonyNetwork(colonyNetworkAddress);
 
+    task.finalized = true;
+
     for (uint8 roleId = 0; roleId <= 2; roleId++) {
       uint payout = task.payouts[roleId][token];
       Role storage role = task.roles[roleId];
 
       uint8 rating = (roleId == EVALUATOR) ? 50 : role.rating;
-      int divider = (roleId == WORKER) ? 30 : 50;
```

Comment:
Why remove this local variable? We can't be running out of stack depth already?

Reply:
Yeah, we are.

Reply:
oh dear..